### DISCUSSION TITLE
Profile mobile version doesnt show the organization

### DISCUSSION THREAD

**Discussion by dtmf7133** [community/community#121644] [Mobile]
### Select Topic Area

Product Feedback

### Body

Im expecting to see my organization in the mobile version of my profile but it doesnt appear, thanks.

**Comment by kangarruu** [comment]
Hi @par7133, thank you for writing in! The Mobile category is actually intended for conversations related to the GitHub Mobile app. I can see 5mode listed under your Mobile App profile so I assume you're referring to the mobile site in a browser. I think you'll have better luck switching the category to `Profile`!  Best of luck!
